_develop/deploy-tips.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: learn
+description: Blockstack app mining documentation
+permalink: /:collection/:path.html
+---
+# How to deploy a Blockstack dApp
+{:.no_toc}
+
+Blockstack applications are a type of web application that uses Blockstack's technology to authenticate users. As such, Blockstack apps tend to be simple in design and operation, since in many cases, they don’t have to host anything besides the application’s assets.
```

Comment:
perhaps phrase this as: "Blockstack applications are web applications that authenticate users with _Blockstack Auth_ and store data with Gaia. Both of these technologies can be accessed on the client side. As such..."